[docs] mention `image` requirement in GH build docs (#27097)
expo/expo: docs/pages/build/building-from-github.mdx

## Prerequisites

### Set the `image` field in your eas.json

> For the build profiles you want to use with GitHub, you need to specify the image to use for the
> build. This is done in the **eas.json** file in your project's root directory. See [the eas.json reference](/eas/json/#image) to learn more.

Reviewer: ```suggestion
For the build profiles you want to use with GitHub, specify an [`image`](/eas/json/#image) to use for the native platform in **eas.json**.

Use the `latest` image if your project's configuration does not rely on a specific [build image](/build-reference/infrastructure/). For example:
```

- Let's make the first line concise and also link to the specific part ("what") of the text instead of mentioning to see the eas.json schema
- Let's also add a description now what images should use in default cases and link to build server infrastructure guide.

```json eas.json
{

>   // ...

Reviewer: ```suggestion
  /* @hide ... */ /* @end */
```

We use this pattern to tell that there is other parts of this config file for concsistency

  "build": {

>     "production": {
>       "ios": {
>         "image": "latest"
>       }
>     }

Reviewer: ```suggestion
    "production": {
     "android": {
        "image": "latest"
      },
      "ios": {
        "image": "latest"
      }
    }
```

Let's also add `android` here because in some cases, this example snippet might bel just copy-pasted and then a question will come arise if the same config required for android too or why it didn't work for android. Specifying both platforms, we try to eliminate the the room for confusion.
